app/tests/unit/pages/howto.spec.js

```diff
@@ -0,0 +1,76 @@
+import { mount, createLocalVue } from '@vue/test-utils'
+import Vue from 'vue'
+import Vuex from 'vuex'
+
```

Comment:
I'm looking at the GitHub checks, and it seems like when the howTo unit tests are run, there are some errors thrown even though the tests pass. I can't tell at first glimpse where they're coming from, but looks like this: 
```"TypeError: The element or ID supplied is not valid. (videojs)"```

Reply:
I noticed that, but I wasn't able to fix it. `videojs` is used by
`v-playback`, and as far as I can tell, it seems like the testing harness is not providing valid
DOM elements for `videojs` to display. It doesn't happen when I open the page in my browser. I'll take another crack at it and see what I can find.

Reply:
I've been digging at this for a while, and I haven't found a solution but I have narrowed down the source of the error. The issue seems to be that in the unit test, the component we used to display video (`v-playback`) is not mounting to the DOM properly. `videojs` queries for `#videoPlayerContainer`, can't find it, and throws that error. 

That confuses me, though. That all happens between lines 37 and 38 below, while on line 38 we query for `#videoPlayerContainer` ourselves and it finds it just fine. I'm pretty stuck on how to continue.

https://github.com/Duke-MatSci/materialsmine/blob/e3f13fc30b9e3090f3cc7fe452a5c9795f1f9851/app/tests/unit/pages/howto.spec.js#L37-L38

Reply:
@roryschadler can we try a pure HTML implementation and remove that video library? See here - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video for implementation. Testing especially when third-party libraries are involved can be frustrating. Let me know if you have questions.

Reply:
Of course. Thanks!